test/models/clip/test_clip.py

```diff
@@ -7,9 +7,9 @@
 import pytest
 import torch
 from test.test_utils import assert_expected, set_rng_seed
-from torchmultimodal.models.clip import CLIP
-from torchmultimodal.modules.encoders.clip_resnet_encoder import ResNetForCLIP
-from torchmultimodal.modules.encoders.clip_text_encoder import CLIPTextEncoder
+from torchmultimodal.models.clip.model import CLIP
+from torchmultimodal.models.clip.resnet_encoder import ResNetForCLIP
```

Comment:
Just wondering, do we wanna just call this image_encoder to be consistent with what we have for other models? (I know Ankita has a PR coming in for CLIP ViT encoder, so maybe we wanna keep things as you have them to keep these in separate files?)

Reply:
maybe we can place both in an `image_encoder.py` file for CLIP, what do you think? cc @ankitade

Reply:
This sounds good to me. Ultimately will leave it up to Ankita as she is working more closely with the CLIP code these days

Reply:
yes image_encoder.py is fine